langchain/src/document.ts

```diff
@@ -1,9 +1 @@
-import { logVersion010MigrationWarning } from "./util/entrypoint_deprecation.js";
```

Comment:
Why drop this?

Reply:
Was logging things when importing from hub due to the `load` call.